code/game/gamemodes/scoreboard.dm

```diff
@@ -121,13 +136,9 @@
 		GLOB.score_crewscore += 2500
 		GLOB.score_powerbonus = 1
 
-	if(GLOB.score_mess == 0)
-		GLOB.score_crewscore += 3000
-		GLOB.score_messbonus = 1
```

Comment:
Correct me if I'm wrong but It looks like this is still used. Line 190:
```
<b>Ultra-Clean Station:</b> [GLOB.score_mess ? "No" : "Yes"] ([GLOB.score_messbonus * 3000] Points)<br><br>
```

Reply:
It is. I removed it from the window text because that bonus only applies if there's NO blood/gibs on the station. At all. Which will occur sometime after the heat death of the universe. I'm hesitant to remove the actual calculation just in case it breaks something else, or I decide to use it again later. Suppose I should just remove the code for now, though.

Reply:
Removed it from the code given it has a .000001% chance of actually happening.